equinix/data_source_network_device.go

```diff
@@ -23,174 +23,6 @@ var deviceStates = []string{ // Not sure if other states should be included
 	// ne.DeviceStateWaitingSecondary,
 }
 
-var ecxNetworkDeviceSchemaNames = map[string]string{
```

Comment:
Since all of this was 100% identical to the definitions in the resource, I redirected all references to those since they're in the same namespace.  In fact many were already cross-referenced and I hadn't noticed - proof that duplicate definitions add confusion.  I took a look around the repo but couldn't find a reasonable analog for this circumstance for inspiration.  I can revert this is you feel strongly duplication is preferred here, but will need to name them something else (again, shared namespace)